.github/workflows/test.yml

```diff
@@ -98,7 +96,7 @@ jobs:
              ghc: '8.10.7'
              test: true
            - os: windows-latest
-             ghc: '9.4.2'
+             ghc: '9.4.4'
```

Comment:
This won't work, unless https://github.com/haskell/actions/issues/129 has been resolved.